client/src/views/ListTeams.vue

```diff
@@ -1,18 +1,20 @@
 <template>
   <div id="userListView" class="container">
     <h3>Teams {{ route.params.year }}</h3>
-    <p v-if="route.params.year == new Date().getFullYear()">
-      Hinweis: Wenn Ihr die Zusammensetzung eures Teams ändern möchtet, tretet
-      bitte direkt in Kontakt mit einem Admin.
-    </p>
-    <div class="row"></div>
-    <button
-      type="button"
-      class="btn btn-outline-primary btn-sm mb-3"
-      @click="onNewTeam"
-    >
-      Melde ein neues Team
-    </button>
+    <div v-if="loggedIn">
```

Comment:
Ein Hinweis, dass man eingeloggt sein muss um ein Team zu melden wäre gut, sonst sucht man vergebens danach.

Und was meinst du damit, dass man sich melden soll wenn man das Team ändern will? Das geht eigentlich nicht. Wenn ein Team geformt ist ist das ein Team für eine Saison. In der nächsten meldet man dann ein neues.

Reply:
Das war noch eine alte Geschichte, da hatte ich mit Sascha drüber gesprochen. Der hatte das wohl so gemacht.
Ich würde es allerdings auch streichen.

Wenn der User es nicht darf, da er nicht eingeloggt ist, würde ich ihm auch die Option nicht anzeigen.

Wenn wir das jetzt wirklich mit dem "Formular"-Reiter machen wollen, dann erübrigt sich die Frage eh von selbst.

Reply:
Jep. Ich hätte dann einen Hinweis angezeigt, dass man angemeldet sein muss um ein Team zu melden. Aber da ich ja eh dafür bin das rauszuschmeißen…;)